ParticleSystem-iOS/GPHParticleView.m

```diff
@@ -5,13 +5,16 @@ @interface GPHParticleView ()
 
 @property (nonatomic, readonly) CAEmitterLayer *emitterLayer;
 @property (nonatomic) NSArray<CAEmitterCell*> *emitterCells;
+@property (nonatomic) float emitterBeginTimeOffset;
+@property (nonatomic) BOOL emitterUseCurrentMediaTime;
 
 @end
 
 @implementation GPHParticleView
 
 @dynamic velocity, scale, lifetime, birthRate, spin, seed, emitterZPosition, preservesDepth;
 @dynamic emitterShape, emitterDepth, emitterMode, renderMode, emitterPosition, emitterSize;
+@dynamic beginTimeOffset, useCurrentMediaTime;
```

Comment:
I don't think these properties can be dynamic since they are not backed by anything thing. The other properties can be dynamic since they are held by the CAEmitterLayer.

So just delete this line.